ResearchKit/Common/ORKLocationSelectionView.m

```diff
@@ -232,27 +234,28 @@ - (void)reverseGeocodeAndDisplay:(CLLocation *)location {
             [strongSelf setAnswer:ORKNullAnswerValue()];
         } else {
             CLPlacemark *placemark = [placemarks lastObject];
-            [strongSelf setAnswer:placemark];
+            [strongSelf setAnswer:[[ORKLocation alloc] initWithPlaceMark:placemark]];
         }
     }];
 }
 
-- (void)setAnswer:(id)answer {
+- (void)setAnswer:(ORKLocation *)answer {
     
     [_mapView removeAnnotations:_mapView.annotations];
     
-    if ([[answer class] isSubclassOfClass:[CLPlacemark class]]) {
-        _answer = [[ORKPlacemark alloc] initWithPlacemark:answer];
-    } else {
+    if (NO == [[answer class] isSubclassOfClass:[ORKLocation class]]) {
```

Comment:
`NO ==` prevent developer accidentally assign it to NO, like  `value == NO` =>  `value = NO`